src/Microsoft.Health.Fhir.Core/Data/OperationDefinition/reindex.json

```diff
@@ -0,0 +1,17 @@
+{
+    "resourceType": "OperationDefinition",
+    "id": "reindex",
+    "url": "[base]/OperationDefinition/reindex",
+    "version": "1.0.0",
+    "name": "Reindex",
+    "title": "Reindex all resources",
+    "status": "active",
+    "kind": "operation",
+    "description": "Reindex all affected resources on the FHIR server with newly added search parameters. This operation is asynchronous as defined in the [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html)",
+    "code": "reindex",
+    "base": "/",
+    "system": true,
+    "type": false,
+    "instance": false,
+    "parameter": []
```

Comment:
We do allow one parameter: maximumConcurrency, use:in, min:1, max:? (we should define this :) ), type: integer

Reply:
Added `maximumConcurrency` to list of parameters for `reindex`. The min and max here determine how many number of times the parameter can appear in the request/response (https://www.hl7.org/fhir/operationdefinition-definitions.html#OperationDefinition.parameter.min). So have set `min: 0` and `max: 1`